storage/aptosdb/src/backup/utils.rs

```diff
@@ -0,0 +1,102 @@
+// Copyright (c) Aptos
+// SPDX-License-Identifier: Apache-2.0
+
+use crate::{
+    change_set::ChangeSet, event_store::EventStore, ledger_store::LedgerStore,
+    schema::transaction_accumulator::TransactionAccumulatorSchema,
+    transaction_store::TransactionStore,
+};
+use anyhow::{ensure, Result};
+use aptos_crypto::HashValue;
+use aptos_types::{
+    contract_event::ContractEvent,
+    ledger_info::LedgerInfoWithSignatures,
+    proof::{definition::LeafCount, position::FrozenSubTreeIterator},
+    transaction::{Transaction, TransactionInfo, Version},
+};
+use schemadb::DB;
+use std::sync::Arc;
+
+/// This file contains utilities that are helpful for performing
+/// database restore operations, as required by db-restore and
+/// state sync v2.
```

Comment:
nit: comments for the whole file better be with "///!" (and maybe brought up before the uses?)

Reply:
nit: maybe call it "restore_utils" to be a bit more explicit.